buildSrc/src/main/kotlin/ai.spotbugs-conventions.gradle.kts

```diff
@@ -19,7 +19,8 @@ spotbugs {
     "Naming",
     "RuntimeExceptionCapture",
     "UnreadFields",
-    "DumbMethodInvocations"
+    "DumbMethodInvocations",
+    "DontReusePublicIdentifiers"
```

Comment:
Violation example:  
`The used identifier ?>?2/2??? as field name in class com.microsoft.applicationinsights.agent.internal.configuration.Configuration$AttributeType.STRING in source file ?>?3/2??? shadows the publicly available identifier from the Java Standard Library.  In Configuration.java`

See https://github.com/spotbugs/spotbugs/blob/4.8.0/CHANGELOG.md

Reply:
The agent class name for example. When one violation is fixed, new ones appear.  We may have plenty of @SuppressWarnings for  this violation type. Not sure this violation really makes sense.